src/jit/protojit/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+project(protojit)
```

Comment:
You should add .gitmirror file on the TFS into this directory first (since it exists on TFS side), wait for it to be mirrored and merged, and then commit this. The mirror is likely going to choke on it otherwise.